src/runtime_src/core/edge/drm/zocl/zocl_bo.c

```diff
@@ -427,16 +428,19 @@ zocl_userptr_bo_ioctl(struct drm_device *dev, void *data, struct drm_file *filp)
 	}
 
 	bo->cma_base.paddr = sg_dma_address((bo->cma_base.sgt)->sgl);
+*/
```

Comment:
Why do we remove this check? What if user use a wrong pointer at the call? We would lost protect.